### PR TITLE
Logging user back in automatically when he gets logged out

### PR DESCRIPTION
**Pull request by henrYoda**

### CODE REVIEW

File: PennMobile/General/UserDefaults + Helpers.swift [hunk 1]
```diff
@@ -508,3 +509,14 @@ extension UserDefaults {
...
+// MARK: - Two Factor Enabled flag
+extension UserDefaults {
+
+    func getTwoFactorEnabledDate() -> Date? {
+        return UserDefaults.standard.value(forKey: UserDefaultsKeys.totpEnabledDate.rawValue) as? Date
+    }
+    
+    func setTwoFactorEnabledDate(_ date: Date?) {
+        UserDefaults.standard.set(date, forKey: UserDefaultsKeys.totpEnabledDate.rawValue)
+    }
```
Comment (joshdoman) [inline PennMobile/General/UserDefaults + Helpers.swift]: Hasn't this already been merged in? Have we not created a PR for the 2FA popup?

File: PennMobile/Auth/TOTPFetcher.swift [hunk 2]
```diff
@@ -46,7 +56,7 @@ class TOTPFetcher: NSObject {
                     WKWebsiteDataStore.createDataStoreWithSavedCookies { (dataStore) in
                         DispatchQueue.main.async {
                             let zombie = WKZombie(dataStore: dataStore)
-                            Logger.enabled = false
+                            Logger.enabled = true
```
Comment (joshdoman) [inline PennMobile/Auth/TOTPFetcher.swift]: Set to false.

**Comment by DominicHolmes** [comment]
What's the status on this PR? @henrYoda @joshdoman

**Comment by joshdoman** [comment]
> What's the status on this PR? @henrYoda @joshdoman

This is basically finished. I'll clean it up based on my comments and merge it in.

**Pull request closed without merging**